tests/api_resources/test_account.py

```diff
@@ -37,36 +37,6 @@ def test_is_saveable(self, request_mock):
         assert isinstance(resource, stripe.Account)
         assert resource is account
 
-    def test_is_saveable_with_additional_owners(self, request_mock):
```

Comment:
I had to remove the test as openapi does not support `legal_entity` anymore. We could mock if needed but I think it's fine to punt on that test.